pom.xml

```diff
@@ -128,6 +131,23 @@ THE SOFTWARE.
           </exclusion>
         </exclusions> 
       </dependency>
+      <dependency>
+	    <groupId>org.apache.ant</groupId>
+	    <artifactId>ant</artifactId>
+	    <version>${ant.version}</version>
+	  </dependency>
+      <dependency>
+	    <groupId>commons-io</groupId>
+	    <artifactId>commons-io</artifactId>
+	    <version>2.5</version>
+	    <scope>runtime</scope>
+	  </dependency>
+      <dependency>
+	    <groupId>xerces</groupId>
+	    <artifactId>xercesImpl</artifactId>
```

Comment:
Xerces dependency should not be needed since Java 7, why is it necessary here?

Reply:
To fix this error but I should find another way:

```
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (display-info) @ jacoco ---
[INFO] Ignoring requireUpperBoundDeps in org.kohsuke:access-modifier-annotation
[WARNING] Rule 6: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for xerces:xercesImpl:2.9.1 paths to dependency are:
+-org.jenkins-ci.plugins:jacoco:2.3-SNAPSHOT
  +-org.apache.maven.reporting:maven-reporting-impl:2.3
    +-org.apache.maven.doxia:doxia-core:1.2
      +-xerces:xercesImpl:2.9.1
and
+-org.jenkins-ci.plugins:jacoco:2.3-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-test-harness:2.32
    +-org.jenkins-ci.main:jenkins-test-harness-htmlunit:2.18-1
      +-xerces:xercesImpl:2.11.0
and
+-org.jenkins-ci.plugins:jacoco:2.3-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-test-harness:2.32
    +-org.jenkins-ci.main:jenkins-test-harness-htmlunit:2.18-1
      +-net.sourceforge.nekohtml:nekohtml:1.9.22
        +-xerces:xercesImpl:2.11.0
```